src/components/banner/banner.less

```diff
@@ -4,54 +4,165 @@
 	position: fixed;
 	right: 0;
 	left: 0;
-	bottom: -1000px;
+	bottom: 0;
 	z-index: 99999;
 
-	display: flex;
-	align-items: center;
-	justify-content: center;
 	color: @color-textLight;
 	background: white;
-	font-size: 18px;
+	font-size: 16px;
 	max-height: 100%;
-	transition: bottom 0.5s ease-in-out;
+	transition: bottom 0.5s ease-in-out, opacity 0.5s linear;
 
 	box-shadow: 0px 0px 5px @color-primary;
-	padding: 0 20px;
+	padding: 15px 30px;
 
-	a {
-		text-decoration: underline;
-		padding: 0 4px;
+	@media @smartphone {
+		margin-top: -80px;
+		padding: 15px 20px;
+		overflow: auto;
 	}
 
-	.title {
-		font-size: 24px;
-		color: @color-text;
-		padding: 20px 0;
-	}
-	.message {
-		padding: 20px 0;
+	&.hidden {
+		pointer-events: none;
+		bottom: -250px;
+		opacity: 0;
 	}
 
-	.expand {
-		padding: 0;
-		height: 20px;
-		fill: @color-primary;
-		transform: rotate(-90deg);
-		transition: 0.5s ease;
+	.content {
+		.message {
+			display: flex;
+
+			@media @smartphone {
+				display: block;
+			}
+
+			.info {
+				flex: 1;
+				line-height: 16px;
+
+				@media @smartphone {
+					font-size: 12px;
+				}
+
+				.title {
+					font-size: 22px;
+					color: @color-text;
+					padding: 15px 0;
+
+					@media @smartphone {
+						font-size: 16px;
+						padding: 5px 0 10px 0;
+					}
+				}
+
+				.options {
+					margin: 15px 0 0 0;
+
+					@media @smartphone {
+						margin: 10px 0 20px 0;
+					}
+
+					.option {
+						margin: 0 20px 6px -2px;
+
+						@media @smartphone {
+							margin: 0 20px 3px -2px;
+						}
+
+						a {
+							text-decoration: none;
+							color: @color-secondary;
+
+							svg {
+								vertical-align: middle;
+								margin-right: 5px;
+								position: relative;
+								top: -1px;
+								transform: rotate(-90deg);
+								transition: 0.5s ease;
```

Comment:
Here's third easing value of `ease`. Could all these separate easings be unified with a global variable?